root/usr/share/nethserver-blacklist/geoip

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#
+# Copyright (C) 2021 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+dir=`/usr/bin/mktemp -d`
+
+/usr/bin/cd $dir
+/usr/bin/wget https://www.ipdeny.com/ipblocks/data/countries/all-zones.tar.gz >/dev/null 2>&1
+tar -xf all-zones.tar.gz
+for j in *.zone
+do
+    echo "#
+# ipv4 hash:ip ipset
+# Category : geoip_blocking
+# Source File Date : $(/usr/bin/date)
+# Maintainer : NethServer
+# List source URL : https://www.ipdeny.com/ipblocks/data/countries/all-zones.tar.gz
+# ISO Country Code : ${j%.zone}
+#
+$(/usr/bin/cat ${j})" > ${j}
+    /usr/bin/mv -- "$j" "ISO_country_code_${j%.zone}.netset"
```

Comment:
Why not creating files directly to the destination dir and avoid the `mv` on line 43?

Also make sure to empty the destination dir before writing down new files.

Reply:
empty folder before to move is ensured now, I would prefer to use a mktemp